server/test/unit/server/managers/consumer/test_agent_manager.py

```diff
@@ -536,6 +543,75 @@ def test_get_distributor():
             self.assertEqual(binding['repo_id'], agent_binding['repo_id'])
             self.assertEqual(None, agent_binding['type_id'])
 
+    @patch('pulp.server.managers.consumer.agent.managers.consumer_manager')
+    @patch('pulp.server.managers.consumer.agent.PulpAgent')
+    def test_delete_queue(self, agent, consumer_manager):
+        url = 'test-url'
+        queue = 'test-queue'
+        consumer_id = 'test-consumer'
+        consumer_manager.return_value.get_consumer.side_effect = MissingResource
+
+        # test
+        AgentManager.delete_queue(url, queue, consumer_id)
```

Comment:
Similar try/except suggested here as [this comment](https://github.com/pulp/pulp/pull/1583/files#r23771860).